Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionTargetDe.kt

```diff
@@ -0,0 +1,129 @@
+package de.rki.coronawarnapp.ui.submission
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.view.accessibility.AccessibilityEvent
+import android.widget.Button
+import android.widget.TextView
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.fragment.app.Fragment
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.databinding.FragmentSubmissionTargetDeBinding
+import de.rki.coronawarnapp.ui.main.MainActivity
+import kotlinx.android.synthetic.main.include_target_de.view.*
+
+/**
+ * Onboarding starting point.
+ */
+class SubmissionTargetDe : Fragment() {
+    companion object {
+        private val TAG: String? = SubmissionTargetDe::class.simpleName
+    }
+
+    private var _binding: FragmentSubmissionTargetDeBinding? = null
+    private val binding: FragmentSubmissionTargetDeBinding get() = _binding!!
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentSubmissionTargetDeBinding.inflate(inflater)
+        return binding.root
+    }
+
+    override fun onDestroyView() {
+        super.onDestroyView()
+        _binding = null
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setButtonOnClickListener()
+    }
+
+    private fun setButtonOnClickListener() {
+        binding.targetDe.targetDeVerify.setOnClickListener {
```

Comment:
please add a comment to a block like this as this is different as the other `fun setButtonClickListener` functions we are using. maybe also think about extracting it to another functions.